tests/helper/helper_oc.go

```diff
@@ -0,0 +1,269 @@
+package helper
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"regexp"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gbytes"
+	"github.com/onsi/gomega/gexec"
+)
+
+type OcRunner struct {
+	// path to oc binary
+	path string
+}
+
+// New initializes new OcRunner
+func NewOcRunner(ocPath string) OcRunner {
+	return OcRunner{
+		path: ocPath,
+	}
+}
+
+// Run oc with given arguments
+func (oc *OcRunner) Run(cmd string) *gexec.Session {
+	session := CmdRunner(cmd)
+	Eventually(session).Should(gexec.Exit(0))
+	return session
+}
+
+// CreateRandProject create new project with random name (10 letters)
+// without writing to the config file (without switching project)
+func (oc *OcRunner) CreateRandProject() string {
+	projectName := "project-" + RandString(8)
+	fmt.Fprintf(GinkgoWriter, "Creating a new project: %s\n", projectName)
+	session := CmdRunner("oc new-project " + projectName + " --skip-config-write")
```

Comment:
the command should use `oc.path` and not hardcoded "oc". It is meant to make sure that this can be used even if `oc` binary is outside of the `PATH`

Reply:
Done